src/controlplane/components.go

```diff
@@ -115,15 +114,15 @@ func findComponentByName(name ComponentName, components []Component) *Component
 	return nil
 }
 
-// labels is a collection of labels, key-value style
-type labels map[string]string
+// componentLabels is a collection of componentLabels, key-value style
+type componentLabels map[string]string
```

Comment:
Should we use "labels.Set{}" or any k8s client object?

Reply:
this will be gone in next PR with new config